### PR TITLE
[DOCFIX] Fix File System API documentation

### PR DESCRIPTION
**Pull request by rongrong**
### What changes are proposed in this pull request?

* Fix javadoc and documentation related to `LocalFirstPolicy` and `LocalFirstAvoidEvictionPolicy`.
* Update link to configuration property `alluxio.user.file.write.tier.default`.
* Update example python code.

### CODE REVIEW

**Review by Xenorith — APPROVED** [review]

File: docs/en/api/FS-API.md [hunk 1]
```diff
@@ -296,8 +288,8 @@ this policy preference exists only for local workers, not remote workers; remote
 blocks to the highest tier.
 
 By default, data is written to the top tier. Users can modify the default setting through the
-`alluxio.user.file.write.tier.default` [configuration]({{ '/en/operation/Configuration.html' | relativize_url }})
-property or override it through an option to the 
+`alluxio.user.file.write.tier.default` [property]({{ '/en/reference/Properties-List.html' | relativize_url }})
```
Comment (rongrong) [inline docs/en/api/FS-API.md:291]: The original link doesn't contain the mentioned property here. It's quite confusing to me why we are linking to the parent page of all configurations here. So I changed it to link to the page that contains this specific property.

File: docs/en/api/FS-API.md [hunk 2]
```diff
@@ -397,74 +389,70 @@ def colorize(code):
...
-    info("done")
-
-    info("asserting that %s is deleted" % py_test_root_dir)
-    assert not client.exists(py_test_root_dir)
-    info("done")
-
-
-if __name__ == '__main__':
-    main()
+py_test_root_dir = '/py-test-dir'
```
Comment (rongrong) [inline docs/en/api/FS-API.md:404]: Code copy pasted from the file referenced above, to be consistent.

File: docs/en/api/FS-API.md (hunk 1 above)
Comment (Xenorith) [inline docs/en/api/FS-API.md:291]: i learned yesterday when reviewing beinan's PR that you can anchor tag every single property in the properties list page -> https://docs.alluxio.io/os/user/stable/en/reference/Properties-List.html#alluxio.user.file.write.tier.default
```suggestion
`alluxio.user.file.write.tier.default` [property]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.user.file.write.tier.default)
```

**Review by Xenorith — APPROVED** [review]
small improvement, otherwise good

**Comment by Xenorith** [comment]
alluxio-bot, merge this please

**Pull request merged**